images/scripts/rhel.setup

```diff
@@ -235,7 +235,7 @@ config_opts['rpmbuild_timeout'] = 86400
 config_opts['yum.conf'] = '[main]\\ncachedir=/var/cache/yum\\ndebuglevel=1\\nlogfile=/var/log/yum.log\\nreposdir=/dev/null\\nretries=20\\nobsoletes=1\\ngpgcheck=0\\nassumeyes=1\\nkeepcache=1\\ninstall_weak_deps=0\\nstrict=1\\n\\n# repos\\n\\n[build]\\nname=build\\nbaseurl=http://download.devel.redhat.com/brewroot/repos/$REPO/latest/x86_64/\\n'
 config_opts['chroot_setup_cmd'] = 'groupinstall build'
 config_opts['target_arch'] = 'x86_64'
-config_opts['root'] = u'rhel-8-candidate-x86_64'
+config_opts['root'] = u'rhel-candidate-x86_64'
```

Comment:
What is this option in mock? From doc I see:
`config_opts['root'] = 'name-of-yum-build-dir'` which is not that clear as one would hope for. But if I understand this correctly this is just to name your build location?

Reply:
Right, I suppose with koji etc. it will appear in some logs and/or URL paths, but for our purposes we don't care.